lib/utils/fs.go

```diff
@@ -84,27 +85,59 @@ func IsDir(path string) bool {
 
 // NormalizePath normalises path, evaluating symlinks and converting local
 // paths to absolute
-func NormalizePath(path string) (string, error) {
+func NormalizePath(path string, evaluateSymlinks bool) (string, error) {
 	s, err := filepath.Abs(path)
 	if err != nil {
 		return "", trace.ConvertSystemError(err)
 	}
-	abs, err := filepath.EvalSymlinks(s)
-	if err != nil {
-		return "", trace.ConvertSystemError(err)
+	if evaluateSymlinks {
+		s, err = filepath.EvalSymlinks(s)
+		if err != nil {
+			return "", trace.ConvertSystemError(err)
+		}
 	}
-	return abs, nil
+	return s, nil
+}
+
+// OpenFileAllowingSymlinks will open a file, if it's a symlink the file referenced by the link will be associated to
+// the returned os.File.  This will return an error if the file is not found or is a directory.
```

Comment:
nit:

```suggestion
// OpenFileAllowingSymlinks opens a file, if it's a symlink the file referenced by the link will be associated to
// the returned os.File.  Returns an error if the file is not found or is a directory.
```

Reply:
Wording updated in a216f6d14848719cd061eb29b1ed657247eaee25